pymemcache/test/test_integration.py

```diff
@@ -232,6 +236,25 @@ def _des(key, value, flags):
 
 
 @pytest.mark.integration()
+def test_serde_serialization(client_class, host, port, socket_module):
+    def check(value):
+        client.set(b'key', value, noreply=False)
+        result = client.get(b'key')
+        assert result == value
+
+    client = client_class((host, port), serializer=python_memcache_serializer,
+                          deserializer=python_memcache_deserializer,
+                          socket_module=socket_module)
+    client.flush_all()
+
+    check(b'byte string')
+    check(u'unicode string')
```

Comment:
Can we also check try some harder cases:
- `'olé'` (with and without specifying the `u` unicode prefix
- list of strings with unicode in them